src/main/kotlin/calculator/DelimiterManager.kt

```diff
@@ -0,0 +1,28 @@
+package calculator
+
+class DelimiterManager {
+
+    fun splitByDelimiter(fixedDelimiter: String?, customInput: String?, input: String): List<String> {
+        return fixedDelimiter?.let { customInput?.split(it.toRegex()) }
+            ?: input.split(DELIMITER.toRegex())
+    }
+
+    fun combineDelimiter(customDelimiter: String?): String? {
+        var fixedDelimiter: String? = null
+        customDelimiter?.let {
+            fixedDelimiter = DELIMITER.plus("|$it")
+        }
+        return fixedDelimiter
+    }
+
+    fun getCustomDelimiter(input: String): String? {
+        val result = Regex("//(.)\n(.*)").find(input)
+        return result?.let {
+            return it.groupValues[1]
+        }
```

Comment:
```suggestion
        return result?.groupValues?.get(1)
```
let을 잘 활용해주셨습니다. 👍 
지금도 좋지만, (?)를 활용하면 한 줄로도 작성할 수 있습니다. 😄

Reply:
let을 사용할 필요가 없었네요! 한줄로 변경하였습니다!